tests/fake_cam.py

```diff
@@ -0,0 +1,27 @@
+from typing import Any, Coroutine, Final, List, Optional, Tuple, Dict
+from viam.components.camera import Camera
+from viam.gen.component.camera.v1.camera_pb2 import GetPropertiesResponse
+from viam.media.video import NamedImage, ViamImage, CameraMimeType
+from viam.media.utils import pil
+from viam.proto.common import ResponseMetadata
+from PIL import Image
+
+
+
+class FakeCamera(Camera):
+
+    def __init__(self, name: str):
+        super().__init__(name=name)
+        self.img = Image.open("tests/dogscute.jpeg")
```

Comment:
You shouldn't use a real image in this case. Use PIL to create a fake image on init since the fake keras model doesn't care what the image is. When possible, do not commit binary files to the git history.

Reply:
It's as simple as `img = Image.new('RGB', (400, 300), color='blue')`